easybuild/easyconfigs/o/OpenBUGS/OpenBUGS-3.2.3-GCC-8.3.0.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author: Guilherme Balzana
+# Research Computing Services - University of Cambridge
+
+easyblock = 'ConfigureMake'
+
+name = 'OpenBUGS'
+version = '3.2.3'
+
+homepage = 'http://openbugs.net'
+description = """BUGS is a software package for performing Bayesian inference
+ Using Gibbs Sampling. The user specifies a statistical model, of (almost)
+ arbitrary complexity, by simply stating the relationships between related
+ variables. The software includes an ‘expert system’, which determines an
+ appropriate MCMC (Markov chain Monte Carlo) scheme (based on the Gibbs sampler)
+ for analysing the specified model. The user then controls the execution of the
+ scheme and is free to choose from a wide range of output types.
+"""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['http://openbugs.net/w/OpenBUGS_3_2_3?action=AttachFile&do=get&target=']
```

Comment:
@guimaluf Please avoid hardcoding the version number in here too, re-use `version` instead.

This should work:

```python
source_urls = ['http://openbugs.net/w/OpenBUGS_%s?action=AttachFile&do=get&target=' % '_'.join(version.split('.'))]
```